doc/user_management.md

```diff
@@ -1,27 +1,123 @@
 
 ```
-application.security.password=$2a$10$MQ5grDaIqDpBjMlG78PFduv.AMRe9cs0CNm/V4cgUubrqdGTFCH3m
+/api/password/{new_password}
+```
+
+Using this endpoint, it is guaranteed that the password strength rules configured in the `application.properties` file will be enforced.
+
+Bare in mind that this endpoint is password protected, and you will have to provide existing credentials in order for TRUE Connector to generate new hash that matches with the value passed in URL, so the general advice is to keep `apiUser` as a kind of administrator account. Once new hash is returned, you can modify properties file and set new password for specific user.
+
+Example:
 
 ```
+# List of users
+users.list=newUser
 
-while apiUser is present in ecc property file. 
+# Credentials for each user
+# encoded - passwordNewUser123
+newUser.password=$2a$12$v7/AKsx5KTNJpwOg9yRRe.h6jP80gc03umqqN6aMAQtvVEWmpIqna
+```
+
+After the files are modified, it is necessary to restart container in order to apply changes.
+
+
+### Modify existing user
+
+Same rules applies to user credentials modification, what it needs to be updated is username in the `user-list`, and if password need to be updated, the same endpoint  should be used. 
+
+
+Example:
 
 ```
-application.user.api.password=$2a$10$MQ5grDaIqDpBjMlG78PFduv.AMRe9cs0CNm/V4cgUubrqdGTFCH3m
+# List of users
+users.list=newUserModify
+
+# Credentials for each user
+# encoded - passwordNewUser123
+newUserModify.password=$2a$12$v7/AKsx5KTNJpwOg9yRRe.h6jP80gc03umqqN6aMAQtvVEWmpIqna
 ```
 
-Both user credentials are not persisted anywhere beside properties files, and their passwords are encoded using BcryptPasswordEncoder.
+Also here, after the files are modified, it is necessary to restart container in order to apply changes.
+
+### Delete existing user
+
+If user needs to be deleted, it can be done by deleting desired username in the in the `user-list`, alongside with assigned password.
+
+Also here, after the files are modified, it is necessary to restart container in order to apply changes.
 
 
-## Modifying password for a user
+## User request for changing password
 
-Once new password is generated, (described [here](https://github.com/Engineering-Research-and-Development/true-connector-execution_core_container/blob/1.14.6/doc/SECURITY.md#change-default-password)) user should send encoded password to the operations user, which should be the only one who can modify connector property file. That user will update property file and restart TRUE Connector, so that new password will be loaded by the connector.
+If user wants to change password previously assigned by admin, it should directly contact the admin, providing new desired password, after which admin will apply changes.
```

Comment:
Admin should inform connector user that password will have to comply with some rules, and that connector user should create new password with those restrictions in mind.